x/farming/types/params.go

```diff
@@ -23,8 +23,11 @@ var (
 	DefaultFarmingFeeCollector    = sdk.AccAddress(address.Module(ModuleName, []byte("FarmingFeeCollectorAcc"))).String()
 	DefaultDelayedStakingGasFee   = sdk.Gas(60000) // See https://github.com/tendermint/farming/issues/102 for details.
 
-	StakingReserveAcc = sdk.AccAddress(address.Module(ModuleName, []byte("StakingReserveAcc")))
 	RewardsReserveAcc = sdk.AccAddress(address.Module(ModuleName, []byte("RewardsReserveAcc")))
+
+	// ReserveAccAddressType is Address type of reserve accounts for staking, rewards, It could be changed,
+	// details on https://github.com/tendermint/farming/issues/200
+	ReserveAccAddressType = AddressType32Bytes
```

Comment:
Which do you think is better, `ReserveAccAddressType` or `ReserveAddressType`?

Reply:
Sure, Let me replace to `ReserveAddressType`